src/libopensc/apdu.c

```diff
@@ -587,9 +587,8 @@ int sc_transmit_apdu(sc_card_t *card, sc_apdu_t *apdu)
 				 *      secure messaging is used */
 				plen          = max_send_size;
 				tapdu.cla    |= 0x10;
-				tapdu.le      = 0;
 				/* the intermediate APDU don't expect data */
```

Comment:
Maybe rewriting the comment would prevent confusion

```suggestion
				/* the intermediate APDU don't expect response data */
```

Reply:
Thanks for the suggestion. Applied!